### ISSUE TITLE
Let match output to nGQL

### ISSUE THREAD

**Issue by Shylock-Hg** [vesoft-inc/nebula#4197]
**Is your feature request related to a problem? Please describe.**

As user need in https://discuss.nebula-graph.com.cn/t/topic/8473

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by czpmango** [comment]
IMHO, it maybe a bad idea bcz the design of ngql is really different from cypher. Some self-consistency issues need to be carefully evaluated.

**Comment by Shylock-Hg** [comment]
> IMHO, it maybe a bad idea bcz the design of ngql is really different from cypher. Some self-consistency issues need to be carefully evaluated.

Yes, but nGQL only accepts a plain variable as input, it's a common concept in current implementation.
